src/test/java/racingcar/Car.kt

```diff
@@ -0,0 +1,19 @@
+package racingcar
+
+class Car(
+    val name: String,
+) {
+    private var _position: Int = 0
+    val position: Int
+        get() = _position
+
+    init {
+        require(name.length <= 5) { "자동차 이름은 5글자를 넘을 수 없습니다." }
+    }
+
+    fun move(movingStrategy: MovingStrategy) {
```

Comment:
```suggestion
    fun move(movingStrategy : MovingStrategy) {
```

ktlint 등의 린트를 미션 과정에 활용해주시면 좋을 것 같네요!

Reply:
@kyucumber 어떤 스타일을 지정해야 제안주신 코드로 포멧팅이 될까요? 기본 설정으로는 안되는 것 같아서 질문드립니다.